app/models/manageiq/providers/ibm_power_hmc/infra_manager/event_target_parser.rb

```diff
@@ -39,7 +41,11 @@ def parse
         # have changed (e.g. RMCState, PartitionName, PartitionState etc...)
         # This may be used to perform quick property REST API calls to the HMC
         # instead of querying the full LPAR data.
-        new_targets << {:assoc => :vms, :ems_ref => elems[:uuid]}
+        if elems[:uuid].eql?(NO_UUID_VALUE)
+          new_targets << ems_event.ext_management_system
```

Comment:
NOTE this works because we have a refresh pre-processor which checks if any of the targets are an EMS, then dropping all other targets and doing a full
https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/base_manager/refresher.rb#L220

Reply:
Actually I took a crack at writing a spec for this, and because the code down here https://github.com/ManageIQ/manageiq-providers-ibm_power_hmc/pull/164/files#diff-fe5f4bef291f246bd6808eda3d3c8d1d8eceb696015b491ff70d1f7784c66a65R61-R68 modifies the targets list we end up with one `InventoryRefresh::Target` that doesn't have an association or a manager_ref